aroma/aroma.py

```diff
@@ -22,17 +26,74 @@ def aroma_workflow(
     TR=None,
     overwrite=False,
     generate_plots=True,
+    debug=False,
+    quiet=False
 ):
+    """Executes The main workflow."""
+    # Create output directory if needed
+    if op.isdir(out_dir) and not overwrite:
+        LGR.info(
+            f"Output directory {out_dir},"
+            """already exists.
+            AROMA will not continue.
+            Rerun with the -overwrite option to explicitly overwrite
+            existing output.""",
+        )
+        return
+    elif op.isdir(out_dir) and overwrite:
+        LGR.warning(
+            "Output directory {} exists and will be overwritten."
+            "\n".format(out_dir)
+        )
+        shutil.rmtree(out_dir)
+        os.makedirs(out_dir)
+    else:
+        os.makedirs(out_dir)
+
+    # Create logfile name
+    basename = 'aroma_'
+    extension = 'tsv'
+    isotime = datetime.datetime.now().strftime('%Y-%m-%dT%H%M%S')
+    logname = os.path.join(out_dir, (basename + isotime + '.' + extension))
+
+    # Set logging format
+    log_formatter = logging.Formatter(
+        '%(asctime)s\t%(name)-12s\t%(levelname)-8s\t%(message)s',
+        datefmt='%Y-%m-%dT%H:%M:%S')
+
+    # Set up logging file and open it for writing
+    log_handler = logging.FileHandler(logname)
+    log_handler.setFormatter(log_formatter)
+    sh = logging.StreamHandler()
+
+    # add logger mode options
+    if quiet:
+        logging.basicConfig(level=logging.WARNING,
+                            handlers=[log_handler, sh],
+                            format='%(levelname)-10s %(message)s')
+    elif debug:
+        logging.basicConfig(level=logging.DEBUG,
+                            handlers=[log_handler, sh],
+                            format='%(levelname)-10s %(message)s')
+    else:
+        logging.basicConfig(level=logging.INFO,
+                            handlers=[log_handler, sh],
+                            format='%(levelname)-10s %(message)s')
+    version_number = _version.get_versions()['version']
+    LGR.info(f'Currently running ICA-AROMA version {version_number}')
     """Run the AROMA workflow.
 
     Parameters
     ----------
     in_feat
     """
-    print("\n------------------------ RUNNING ICA-AROMA ------------------------")
-    print("-------- 'ICA-based Automatic Removal Of Motion Artifacts' --------\n")
+    LGR.info("\n------------------------ "
+             "RUNNING ICA-AROMA ------------------------")
+    LGR.info("-------- "
+             "'ICA-based Automatic Removal Of Motion Artifacts' --------\n")
```

Comment:
Do we really want this? I mean, anyone using this package will know they're using ICA-AROMA. I'd remove these lines.

Reply:
They were already in the code